ssz_rs/src/merkleization/cache.rs

```diff
@@ -12,7 +12,7 @@ impl Cache {
     pub fn with_leaves(leaf_count: usize) -> Self {
         Self {
             leaf_count,
-            dirty_leaves: bitvec![1; leaf_count],
+            dirty_leaves: bitvec![1,],
```

Comment:
does this make a bitvector of length `leaf_count` that has all bits set?

Reply:
Now updated (055129e) to explicitly define bitvec length to be equal to leaf_count.